python/mlir/tpp/sched/bundles.py

```diff
@@ -34,7 +33,12 @@ def tpp_mapping(mod, lower_pack_unpack_without_transpose: bool = False, **_confi
     func = apply_registered_pass(func, "pack-conv2DNchwFchw")
     func = apply_registered_pass(func, "pack-conv2DNhwcHwcf")
     func = apply_registered_pass(func, "rewrite-conv-to-matmul-or-brgemm")
-    func = apply_registered_pass(func, "pack-matmul")
+    m = select("m", [2, 4, 8])
+    n = select("n", [4, 8, 16])
+    k = select("k", [2, 4, 8, 16])
```

Comment:
I'd suggest better default values, at least the ones that get picked by the default `select` logic.

Reply:
Have now made it so that the user is responsible for passing in options, including multiple values in case `tpp-tune` or the like is supposed to resolve the choices.

Reply:
This branch/PR was mostly meant as a demo that I wanted to share. Will keep on updating it until it's ready to go into main. Have now marked the PR draft until its ready for proper review.